content/en/blog/_posts/2022-04-07-Kubernetes-1-24-removals-and-deprecations.md

```diff
@@ -69,7 +69,7 @@ been deprecated. These removals have been superseded by newer, stable/generally
 
 * [Dynamic kubelet configuration](https://github.com/kubernetes/enhancements/issues/281): `DynamicKubeletConfig` is used to enable the dynamic configuration of the kubelet. The `DynamicKubeletConfig` flag was deprecated in Kubernetes 1.22. In v1.24, this feature gate will be removed from the kubelet. See [Reconfigure kubelet](/docs/tasks/administer-cluster/reconfigure-kubelet/). Refer to the ["Dynamic kubelet config is removed" KEP](https://github.com/kubernetes/enhancements/issues/281) for more information. 
 * [Dynamic log sanitization](https://github.com/kubernetes/kubernetes/pull/107207): The experimental dynamic log sanitization feature is deprecated and will be removed in v1.24. This feature introduced a logging filter that could be applied to all Kubernetes system components logs to prevent various types of sensitive information from leaking via logs. Refer to [KEP-1753: Kubernetes system components logs sanitization](https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/1753-logs-sanitization#deprecation) for more information and an [alternative approach](https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/1753-logs-sanitization#alternatives=).
-* In-tree provisioner to CSI driver migration: This applies to a number of in-tree plugins, including [Portworx](https://github.com/kubernetes/enhancements/issues/2589). Refer to the [In-tree Storage Plugin to CSI Migration Design Doc](https://git.k8s.io/design-proposals-archive/storage/csi-migration.md#background-and-motivations) for more information. 
+* In-tree provisioner to CSI driver migration: This applies to a number of in-tree plugins, including [Portworx](https://github.com/kubernetes/enhancements/issues/2589). Refer to the [In-tree Storage Plugin to CSI Migration Design Doc](https://git.k8s.io/design-proposals-archive/storage/csi-migration.md#background-and-motivations) for more information. _Please note that the following content was written with the assumption that [Portworx](https://git.k8s.io/design-proposals-archive/storage/csi-migration.md#background-and-motivations) will be able to target Beta in 1.24. However, [Portworx](https://git.k8s.io/design-proposals-archive/storage/csi-migration.md#background-and-motivations) didn't make it, so it stayed in Alpha in 1.24.There are several practices going on with [Portworx](https://git.k8s.io/design-proposals-archive/storage/csi-migration.md#background-and-motivations) to target Beta again in 1.25._
```

Comment:
The scope of #34144 is to remove the section, rather than replace it. The replacement text provides more detail than seems helpful. 

Additionally, the style guide standard is to [use present tense](https://kubernetes.io/docs/contribute/style/style-guide/#use-simple-and-direct-language).

@NitishKumar06 I think this PR can work if you remove the line rather than replace it.

Reply:
@zacharysarah Do you want to remove the whole paragraph at line 72 or just the new changes only that I've added?

Reply:
+1 to @zacharysarah 's [comment](https://github.com/kubernetes/website/pull/34281#discussion_r899826308)
> The scope of https://github.com/kubernetes/website/issues/34144 is to remove the section, rather than replace it. The replacement text provides more detail than seems helpful.
> 
> Additionally, the style guide standard is to [use present tense](https://kubernetes.io/docs/contribute/style/style-guide/#use-simple-and-direct-language) and [simple, direct language](https://kubernetes.io/docs/contribute/style/style-guide/#use-simple-and-direct-language).

Please remove the following section:
> In-tree provisioner to CSI driver migration: This applies to a number of in-tree plugins, including [Portworx](https://github.com/kubernetes/enhancements/issues/2589). Refer to the [In-tree Storage Plugin to CSI Migration Design Doc](https://github.com/kubernetes/design-proposals-archive/blob/main/storage/csi-migration.md#background-and-motivations) for more information.